freetextresponse/public/view.js

```diff
@@ -74,6 +84,22 @@ function FreeTextResponseView(runtime, element) {
         return false;
     });
 
+    function htmlBuilder(responses) {
+        /*
+        Convert list of responses to a html string to add to the page
+        */
+        var html = '';
+        //for each item in the list, add a div containing it to the html
+        responses.forEach(function(item) {
+            html += '<div class="other-student-responses">' + item.answer + '</div>';
+        });
+        //if the answer list is empty, add the appropriate string to the page
+        if (html === '') {
+            html = '<p class="no-response"> No responses to show at this time </p>';
```

Comment:
if you default `response-list` to have this HTML in it, and replace only when there is content you can simplify this a lot.